packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -268,10 +285,9 @@ export class MarketDB extends DerivedDB {
               longDescription = info.longDescription ? info.longDescription : "";
               resolutionSource = info.resolutionSource ? info.resolutionSource : "";
               _scalarDenomination = info._scalarDenomination ? info._scalarDenomination : "";
-              tags = info.tags ? info.tags.toString() : ""; // convert to comma separated so it is searchable
             }
 
-            this.flexSearch.add({
+            this.flexSearchIndex.add({
```

Comment:
below is prob where my comment should have been. what if some of the field values are empty like "" or undefined does that string get indexed?

Reply:
It defaults to being set to empty string and then gets set to the category if it's specified. I don't think indexing the empty string here should be an issue.